### ISSUE TITLE
repair station

### ISSUE THREAD

**Issue by gmyuval** [starwards/starwards#547]
repair widgets / minigames.
preferably behind a network API (IoT?)

blocked by #1228

**Comment by amir-arad** [comment]
MVP to be developed in node-red using dashboard.

each key selects a different system to fix, each "fix" is a simple progress bar.

I've made [https://flows.nodered.org/node/@amir-arad/node-red-countdown](this node) to enable the countdown logic in node-red.

**Comment by amir-arad** [comment]
blocked by #1228

**Comment by amir-arad** [comment]
Blocker #1228 (malfunction API) is closed. This can be unblocked.

Current design: 3-tier repair system (field repair / docked repair / shipyard). See starwards-design/mechanics/armor-and-damage.md. Deferred to post-first-event per the "ship early, expand later" strategy.

**Comment by amir-arad** [comment]
Backlog triage (2026-07-19): moved to the design KB — https://github.com/starwards/starwards-design/blob/main/product/backlog.md#ship-content-scenario--world

GitHub issues are now dispatch tickets only ([ADR-0003](https://github.com/starwards/starwards-design/blob/main/governance/decisions/0003-github-as-dispatch-only.md)). To pick this up again, open a fresh dispatch issue from the backlog entry (don't reopen this one).
